src/resources/postcss/admin/notice-install-event-tickets.pcss

```
@@ -0,0 +1,32 @@
+/* The Events Calendar Utilities */
+@import '../utilities/_all.pcss';
+
+.tribe-notice-event-tickets-install,
+.tribe-notice-event-tickets-activate {
+	border-left-color: var(--tec-color-accent-primary);
+	border-radius: 4px;
+	display: flex;
+	padding: 0;
+
+	h3 {
+		line-height: 1.2;
```

```suggestion
		line-height: var(--tec-line-height-0); 
```
That's 1.38 - the smallest we use in our FE common styles;